docker/config/dynamicconfig/development_es.yaml

```diff
@@ -0,0 +1,29 @@
+frontend.enableClientVersionCheck:
```

Comment:
can we copy it from the /config/dynamicconfig? So that we don't have to maintain two copies. Because this file is changing fast.

Reply:
Sure